util/dockerfiles/docker-bake.hcl

```diff
@@ -35,59 +35,147 @@ variable "TAG" {
   default = "latest"
 }
 
+# Common attributes across all targets. Note: these can be overwritten.
+target "common" {
+  # Here we are enabling multi-platform builds. We are compiling to both ARM
+  # amd X86.
```

Comment:
This comment doesn't match the next line which also has RISC-V.

Does it make sense to build these dockerfile images for RISC-V? Right now, I don't know of anyone with RISC-V hardware fast enough to run and test gem5.

Reply:
I included it because it really costs us next-to-nothing. It's not a big extra step and I quite like supporting all three in theory. Yes, not high utility for now, but's there.

Reply:
I updated the comment.